docs/usage.md

```diff
@@ -215,47 +222,41 @@ For example:
 --split_fastq 10000
 ```
 
-### --sample
-
-> :warning: This params is deprecated -- it will be removed in a future release.
-> Please check: [`--input`](#--input)
-
-### --sampleDir
+### --trim_fastq
+Use this to perform adapter trimming [Trim Galore](https://github.com/FelixKrueger/TrimGalore/blob/master/Docs/Trim_Galore_User_Guide.md)
 
-> :warning: This params is deprecated -- it will be removed in a future release.
-> Please check: [`--input`](#--input)
+### --clip_r1
+Instructs Trim Galore to remove <int> bp from the 5' end of read 1 (or single-end reads). This may be useful if the qualities were very poor, or if there is some sort of unwanted bias at the 5' end.
 
-### --annotateVCF
+### --clip_r2
+Instructs Trim Galore to remove <int> bp from the 5' end of read 2 (paired-end reads only). This may be useful if the qualities were very poor, or if there is some sort of unwanted bias at the 5' end.
 
-> :warning: This params is deprecated -- it will be removed in a future release.
-> Please check: [`--input`](#--input)
+### --three_prime_clip_r1
+Instructs Trim Galore to remove <int> bp from the 3' end of read 1 (or single-end reads) AFTER adapter/quality trimming has been performed. This may remove some unwanted bias from the 3' end that is not directly related to adapter sequence or basecall quality.
 
-Multiple VCF files can be specified if the path must be enclosed in quotes
+### --three_prime_clip_r2
+Instructs Trim Galore to re move <int> bp from the 3' end of read 2 AFTER adapter/quality trimming has been performed. This may remove some unwanted bias from the 3' end that is not directly related to adapter sequence or basecall quality.
 
-### --no_gvcf
+### --trim_nextseq
+This enables the option --nextseq-trim=3'CUTOFF within Cutadapt, which will set a quality cutoff (that is normally given with -q instead), but qualities of G bases are ignored. This trimming is in common for the NextSeq- and NovaSeq-platforms, where basecalls without any signal are called as high-quality G bases. 
 
-Use this to disable g.vcf from `HaplotypeCaller`.
+### --save_trimmed
+Option to keep trimmed fastqs
 
-### --noGVCF
+### --sample
 
 > :warning: This params is deprecated -- it will be removed in a future release.
-> Please check: [`--no_gvcf`](#--no_gvcf)
-
-### --skip_qc
-
-Use this to disable specific QC and Reporting tools.
-Available: `all`, `bamQC`, `BCFtools`, `FastQC`, `MultiQC`, `samtools`, `vcftools`, `versions`
-Default: `None`
+> Please check: [`--input`](#--input)
 
-### --skipQC
```

Comment:
Just noticed that this `--skipQC` section has disappeared from the docs as well.